python/lsst/ap/pipe/ap_pipe.py

```diff
@@ -208,6 +208,13 @@ def parsePipelineArgs():
     parser.add_argument('-i', '--dataId',
                         help="Butler identifier naming the data to be processed (e.g., visit and ccdnum) \
                               formatted in the usual way (e.g., 'visit=54321 ccdnum=7').")
+    templateFlags = parser.add_mutually_exclusive_group()
+    templateFlags.add_argument('--templateId', default='visit=410929',
+                               help="A Butler identifier naming a visit to use as the template \
+                                     (e.g., 'visit=101'). Defaults to 'visit=410929' if neither \
+                                     --templateId nor --templateRepo provided.")
+    templateFlags.add_argument('-t', '--templateRepo',
+                               help="A URI to a Butler repository that will be searched for coadd templates")
```

Comment:
I would say here that if templateRepo is provided templateId will be ignored (as in the code below)

Reply:
What are you suggesting I say, exactly? `ArgumentParser` will already mark the two arguments as mutually exclusive.

Reply:
Does `ArgumentParser` stop execution if you specify both?

Reply:
Yes

Reply:
Okay, than I withdraw my concern here.